src/Orleans/Logging/LoggerImpl.cs

```diff
@@ -240,13 +240,12 @@ private void WriteLogMessage(int errorCode, Severity sev, string format, object[
             {
                 try
                 {
-                    consumer.Log(sev, loggerType, Name, message, LogManager.MyIPEndPoint, exception, errorCode);
+                    consumer.Log(sev, Name, message, exception, errorCode);
```

Comment:
The following PR will be replacing Orleans legacy infra with new ASP.net abstraction